packages/react-native-web/src/modules/useHover/index.js

```diff
@@ -81,8 +81,8 @@ export default function useHover(targetRef: any, config: HoverEventsConfig): voi
           onHoverChange(false);
         }
         // Remove the listeners once finished.
-        addMoveListener(target, null);
-        addLeaveListener(target, null);
+        addMoveListener && addMoveListener(target, null);
```

Comment:
We shouldnt need to guard these calls, and guards should always use if != null statements instead

Reply:
This was something I tried to note, but my relative lack of hook experience may show here. It does actually look from a JS perspective like it could be null (`useEvent` appears to nullify the listener and return it on cleanup, so technically `useEvent` can return null). I know in practice that's unlikely to happen though